README.md

```diff
@@ -105,8 +111,57 @@ Login at [http://localhost:3000](http://localhost:3000) with *`user@example.com`
   ```
   docker-compose -f docker-compose.dev.yml up -d
   ```
+Login at [http://localhost:3000](http://localhost:3000) with *`user@example.com`*/*`changeme`*.
+
+Or if you want to use Oracle as source for your HOMS instance
+
+##### With Oracle Instant Client
+
+Steps 1 – 6 are the same as for [without Oracle Instant Client installation](#without-oracle-instant-client)
+
+7. Download the Oracle Instant Client 11.2 archives from OTN:
+
+http://www.oracle.com/technetwork/topics/linuxx86-64soft-092277.html
```

Comment:
I'm not sure, but this link may become invalid after a few months. Use some general link: http://www.oracle.com/technetwork/database/database-technologies/instant-client/downloads/index.html or something like that.

Reply:
I wasn't sure too, but I found [a lot of snaphots](https://web.archive.org/web/*/http://www.oracle.com:80/technetwork/topics/linuxx86-64soft-092277.html) of this page with the same URL from 28 Aug 2010 in webarchive.
Maybe it rotates, but it would be strange, seems to me.

Reply:
Ok